src/program.cpp

```diff
@@ -546,6 +553,81 @@ std::vector<argument> program::eval(parameter_map params) const
     }
 }
 
+const int program_file_version = 1;
```

Comment:
Can we use this variable to indicate the migraphx version as in issue #186

Reply:
No, this is the version of the file not for migraphx. If we change the format, then we would need to bump the version.